semgrep/tests/e2e/snapshots/test_debugging_json/test_debugging_json/results.json

```diff
@@ -0,0 +1,310 @@
+{
+  "debug": [
+    {
+      "rules.assert-eqeq-is-ok": [],
+      "rules.eqeq-is-bad": [
+        {
+          "filter": "initial",
+          "metavar_ranges": {},
+          "pattern_id": null,
+          "ranges": {
+            ".0": [
+              [
+                72,
+                174,
+                {}
+              ]
+            ],
+            ".2": [
+              [
+                249,
+                267,
+                {}
+              ]
+            ],
+            ".4.4.0": [
+              [
+                260,
+                266,
+                {
+                  "$X": "def60a1f54243f39909595046ac2663d"
+                }
+              ],
+              [
+                168,
+                174,
+                {
+                  "$X": "def60a1f54243f39909595046ac2663d"
+                }
+              ],
+              [
+                55,
+                69,
+                {
+                  "$X": "8136864be65be7e901b2dfd746807f2a"
+                }
+              ]
+            ]
+          }
+        },
+        {
+          "filter": "pattern-not-inside",
+          "metavar_ranges": {},
```

Comment:
These were the only changes in the snapshot, before they were conditionally included per-step.